### PR TITLE
Use TokenType from golibs

### PR DESCRIPTION
**Pull request by vikmeup**

### CODE REVIEW

**Review by arslan-raza-143 — COMMENTED** [review]
- [ ] @vikmeup

File: platform/ethereum/trustray/token.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package trustray
 
 import (
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
```
Comment (arslan-raza-143) [inline platform/ethereum/trustray/token.go:4]: @arslan-raza-143

File: platform/ethereum/blockbook/token.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package blockbook
 
 import (
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
```
Comment (arslan-raza-143) [inline platform/ethereum/blockbook/token.go:4]: @arslan-raza-143

File: platform/ethereum/blockbook/token.go [hunk 3]
```diff
@@ -31,6 +32,6 @@ func NormalizeToken(srcToken *Token, coinIndex uint) blockatlas.Token {
 		TokenID:  srcToken.Contract,
```
Comment (arslan-raza-143) [inline platform/ethereum/blockbook/token.go:31]: - [ ] [](https://www.amazon.com/a/c/r/mvllpHnlBio8ieuTtU0pitEx4)

**Review by arslan-raza-143 — COMMENTED** [review]
-

**Review by arslan-raza-143 — COMMENTED** [review]
- [ ]

**Review by Hjordaan2002 — APPROVED** [review]

**Pull request merged**